src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web/Controllers/DraftApprenticeshipController.cs

```diff
@@ -34,15 +37,17 @@ public class DraftApprenticeshipController : Controller
         private readonly ICommitmentsApiClient _commitmentsApiClient;
         private readonly IModelMapper _modelMapper;
         private readonly IEncodingService _encodingService;
+        private readonly IFeatureTogglesService<ProviderFeatureToggle> _featureTogglesService;
 
         public const string DraftApprenticeDeleted = "Apprentice record deleted";
 
-        public DraftApprenticeshipController(IMediator mediator, ICommitmentsApiClient commitmentsApiClient, IModelMapper modelMapper, IEncodingService encodingService)
+        public DraftApprenticeshipController(IMediator mediator, ICommitmentsApiClient commitmentsApiClient, IModelMapper modelMapper, IEncodingService encodingService, IFeatureTogglesService<ProviderFeatureToggle> featureTogglesService)
```

Comment:
Looking into this the service which should be used is SFA.DAS.Authorization.Services.IAuthorizationService as the IFeatureToggle.... is more specific to the Provider context. I think the use of this service in the cohort page was incorrect.